.github/actions/deploy/deploy.py

```
@@ -0,0 +1,696 @@
+                    self.deploy_pypi_server()
+
+                # Deploy DSP directly
+                self.deploy_dsp_direct()
+
+            # Setup port forwarding
+            self.forward_port()
+
+            print('🎉 Deployment completed successfully!')
+
+        except Exception as e:
+            print(f'❌ Deployment failed: {str(e)}')
```

Ruff (0.14.10)

Use explicit conversion flag

Replace with conversion flag

(RUF010)